js/private/link_js_package.bzl

```diff
@@ -14,17 +15,16 @@ _LinkJsPackageInfo = provider(
         "version": "version of this node package",
         "dep_refs": "list of dependency ref targets",
         "virtual_store_directory": "the TreeArtifact of this node package's virtual store location",
-        "linked_js_package_dir": "the symlink of this package at the root of the node_modules if this is a direct npm dependency",
     },
 )
 
 _ATTRS = {
     "src": attr.label(
-        allow_single_file = True,
-        doc = """A source directory or TreeArtifact containing the package files.
+        doc = """A js_package target or a target that provides a JsPackageInfo.
```

Comment:
nit: they're not mutually exclusive so "or any other target that provides"

Reply:
:taco: